package.json

```diff
@@ -21,33 +21,40 @@
   "scripts": {
     "coverage": "istanbul cover _mocha -x src/model.js --  --compilers js:babel-core/register --recursive -r jsdom-global/register",
     "start": "node example/server.js",
-    "test": "mocha --compilers js:babel-core/register --recursive -r jsdom-global/register",
+    "test": "eslint . && mocha --compilers js:babel-core/register --recursive -r jsdom-global/register",
```

Comment:
this is a package I am using for that:

https://www.npmjs.com/package/pre-push